### DISCUSSION TITLE
Возможность отвязать гитхаб

### DISCUSSION THREAD

**Discussion by Kukushechka7-ezx** [Sokol-Games-GmbH/PixelCombats.PublicApi#79] [Q&A]
Сделать возможность отвязать аккаунт gitHub.

**Comment by forwork10** [comment]
не нужно) в любой момент можно привязать другой гитхаб...

**Comment by Kukushechka7-ezx** [comment]
Невозможно, лол

**Comment by forwork10** [comment]
ты пробовал?) Ты понимаешь что я гамного  превосходу тебя и пытаешься сделать себя всегда правым...

**Comment by Kukushechka7-ezx** [comment]
Может, потому что ты написал ЗОМБИ? Ахаххаха

**Comment by forwork10** [comment]
может потому что ты олух!

**Comment by Kukushechka7-ezx** [comment]
Слит

**Comment by kkohno** [comment]
а смысл его отвязывать?

**Comment by Kukushechka7-ezx** [comment]
Чтобы новый привязать

**Comment by forwork10** [comment]
его изи отвязать кстати)) ты просто тупой

**Reply by kkohno** [reply]
и как его отвязать? Опишите, пожалуйста

**Reply by forwork10** [reply]
в настройках аккаунта гитхаб можно
